cli/status/status.go

```diff
@@ -39,6 +40,23 @@ func Status(runningCtx running.RunningCtx) error {
 		if procStatus.Code == process_utils.ProcessRunningCode {
 			fmt.Fprintf(tw, "%d", procStatus.PID)
 		}
+
+		conn, err := connector.Connect(connector.ConnectOpts{
```

Comment:
Can we set the timeout for the connection explicitly? To 1 second for example. This way, the status won't take too long to print.

Reply:
Timeout is hardcoded in `func Connect(opts ConnectOpts) (Connector, error)`:
```go
greetingConn.SetReadDeadline(time.Now().Add(greetingOperationTimeout))
greetingOperationTimeout = 3 * time.Second
```